.github/actions/setup-matrix/index.js

```diff
@@ -0,0 +1,19 @@
+const core = require('@actions/core');
+
+try {
+    // TODO remove 3 once getInput gets fixed
+    // https://github.com/actions/toolkit/issues/1576
+    const numShards = core.getInput('shards') || 3
+    const shards = []
+
+    for (let i = 1; i <= numShards; i++) {
+        shards.push({
+            number: i
```

Comment:
Related to https://github.com/OpenPhone/gha/pull/89/commits/06233c447e30cb5d3a5fdd7b7c500412ad2d90cc#r1379401046

Would we want to push `.push(`${i}/${numShards}`)